src/features/edit-nonce-drawer/edit-nonce-drawer.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useCallback } from 'react';
+import { Stack } from '@stacks/ui';
+
+import { useDrawers } from '@common/hooks/use-drawers';
+import { LoadingKeys, useLoading } from '@common/hooks/use-loading';
+import { openInNewTab } from '@common/utils/open-in-new-tab';
+import { ControlledDrawer } from '@components/drawer/controlled';
+import { Link } from '@components/link';
+import { Caption } from '@components/typography';
+import { useShowEditNonceCleanupEffect } from '@store/ui/ui.hooks';
+
+import { EditNonceForm } from './components/edit-nonce-form';
+
+const URL = 'https://www.hiro.so/questions/transactions-advanced-settings';
+
+const CustomFeeMessaging = () => {
+  return (
+    <>
+      <Caption>
+        If your transaction has been pending for a long time, its nonce might not be correct.{' '}
+        <Link fontSize="14px" onClick={() => openInNewTab(URL)}>
+          Learn more.
+        </Link>
+      </Caption>
+    </>
```

Comment:
```suggestion
    <Caption>
      If your transaction has been pending for a long time, its nonce might not be correct.{' '}
      <Link fontSize="14px" onClick={() => openInNewTab(URL)}>
        Learn more.
      </Link>
    </Caption>
```

Reply:
Out of curiosity: Is the font size hardcoded at 14px?

Reply:
I just renamed this file so this isn't my code. The `Link` component does take a hardcoded fontSize as a prop and it looks like the default is hardcoded to `12px`.